src/human-readable/types/structs.ts

```diff
@@ -1,9 +1,9 @@
-import type { AbiParameter } from '../../abi.js'
+import type { InferredAbiParameter } from '../../abi.js'
```

Comment:
Had to change this to `InferredAbiParameter` because while parsing structs with `StrictAbiType = true` it would cause some components to be `never`.